src/LocationsAdmin/NewLocationForm.tsx

```diff
@@ -89,6 +141,52 @@ function NewLocationForm() {
             type="number"
           />
         </Form.Item>
+        <Form.Item
+          requiredMark={true}
+          label="Location Format" required
+          tooltip="Select the location format acording documentation"
+          name="format"
+          rules={[{ required: true, message: 'This can not be empty 🤦' }]}>
+          <Select
+            placeholder="Select the format"
+            onChange={value => { handleFormatChange(value?.toString()); }}
+          >
+            <Option value="billboard">Billboard</Option>
+            <Option value="mini-billboard">Mini Billboard</Option>
+            <Option value="digital">Digital</Option>
+          </Select>
+        </Form.Item>
+        <Form.Item
+          requiredMark={true}
+          label="Location Orientation" required
+          tooltip="Select the location orientation acording documentation"
+          name="orientation"
+          rules={[{ required: true, message: 'This can not be empty 🤦' }]}>
+          <Select
+            placeholder="Select the orientation"
+            onChange={value => { hanldeOrientationChange(value?.toString()); }}
```

Comment:
Here as well: This method should use the correct value instead of doing `value?.toString()`